_posts/2020-03-26-CollegeBreaks.md

```diff
@@ -0,0 +1,66 @@
+author: Arka
+categories: [ College Breaks, Getting Started, Time Utilization]
+image: assets/images/CollegeBreak.jpg
+last_added: true
+hidden: false
+---
+![](https://github.com/monsij/insight/raw/master/assets/images/mainpic.jpg)
+
+##Introduction :
+
+After our semester examinations end, we return home for a month or two. Many of the students have already planned everything about how they are going to use this time judiciously, while many are completely in the dark. This article is meant to provide you with some ideas as to make good use of this vital time in your college lives.
+
+
+## Create and update your LinkedIn account :
+![](https://github.com/monsij/insight/raw/master/assets/images/linkedin.png)
+
+This vacation, remember to update your LinkedIn profile. LinkedIn is a very important platform for professionals and students. Many companies skim through LinkedIn for prospective employees and many job profiles require a LinkedIn account just to apply. Also don't forget to upload a professional profile picture.
+
+
+##Make or Update your resume :
+![](https://github.com/monsij/insight/raw/master/assets/images/resume.png)
+
+This is a great time to update your resume with all of the experience and skills you have inculcated during your journey. Mention your fields of interest, your accomplishments and all the important information adding up to a good CV. Keep it short, keep it sweet! If you don't have a resume, create one now! It is never too early to have one. Some great free websites for making catchy resumes are resumake.io, flowcv.io and enhancv.com.
+
+
+##Look for Internships:
+![](https://github.com/monsij/insight/raw/master/assets/images/intern.png)
+
+This is the time when you should look for internship opportunities. Internships are a great way to gain a valuable job or research experience, along with some handy money as well. Go through all the internship options being offered to complement your field of interest and start applying, but do not forget to make a good resume before doing so! A resume plays a big role in the first impression the recruiter has of you.
+
+
+##Sell or Exchange old books:
+![](https://github.com/monsij/insight/raw/master/assets/images/books.png)
+
+Are you one among the book lovers?
+You can utilize this time to sell old books that you don’t need anymore. You’ll gain valuable real estate while also getting some money. Apart from selling you can exchange old books among your friends and relatives and have a good reading experience.
+
+
+##Do some online courses:
+![](https://github.com/monsij/insight/raw/master/assets/images/course.png)
+
+Let’s be honest, most of us are hardly learning anything in college classrooms while struggling with boring curriculums. This is the time to learn things that you couldn’t understand in class, or even topics which are not there in the curriculum. Websites like Udemy, Coursera, edX, Udacity and Pluralsight are some of the great for learning.
```

Comment:
1. Title : *Take up online courses*
2. Comma after "class" is not required. 
3. ~which are~ -> that are
4. great **platforms** for **taking up new skills**.